src/sections/Community/Handbook/repository.js

```diff
@@ -143,6 +148,15 @@ const Repository = () => {
                             </td>
                             <td>{project}</td>
                             <td>{language}</td>
+                            <td>
+                              {maintainers_name.map((mname, index) => {
+                                return (
+                                  <a href={link[index]} rel="noreferrer" target="_blank" key={index}>
```

Comment:
Since all are internal links, use the gatsby `Link` component

Reply:
sounds good , done :+1: